app/views/soft-close/alert.html

```diff
@@ -0,0 +1,32 @@
+{% extends "_wizard-form.html" %}
+{% set title = 'Registration has temporarily closed.' %}
```

Comment:
```suggestion
{% set title = 'Registration has temporarily closed' %}
```
We do not need full stops in titles here